### PR TITLE
Changing injection of SecureRandom

### PR DESCRIPTION
**Pull request by dmelosantos**

### CODE REVIEW

File: dist/cjs/creds/VerifiableCredential.js [hunk 1]
```diff
@@ -164,6 +162,7 @@ class CvcMerkleProof {
     const currentLength = nodes.length;
     const targetLength = currentLength < CvcMerkleProof.PADDING_INCREMENTS ? CvcMerkleProof.PADDING_INCREMENTS : _.ceil(currentLength / CvcMerkleProof.PADDING_INCREMENTS) * CvcMerkleProof.PADDING_INCREMENTS;
     const newNodes = _.clone(nodes);
+    const secureRandom = services.container.SecureRandom;
```
Comment (jpsantosbh) [inline dist/cjs/creds/VerifiableCredential.js]: Is not better to follow the same pattern as getAnchorService()?

File: dist/cjs/uca/UserCollectableAttribute.js [hunk 2]
```diff
@@ -166,6 +165,7 @@ const parseAttestableValue = value => {
 function UCABaseConstructor(identifier, value, version) {
   this.timestamp = null;
   this.id = null;
+  this.secureRandom = services.container.SecureRandom;
```
Comment (jpsantosbh) [inline dist/cjs/uca/UserCollectableAttribute.js]: Is not better to follow the same pattern as getAnchorService()? Same as before

Comment (jpsantosbh) [reply dist/cjs/uca/UserCollectableAttribute.js]: Do we need to attach the secure handler to this? (An UCA instance)

File: dist/es/creds/VerifiableCredential.js [hunk 3]
```diff
@@ -160,6 +158,7 @@ class CvcMerkleProof {
     const currentLength = nodes.length;
     const targetLength = currentLength < CvcMerkleProof.PADDING_INCREMENTS ? CvcMerkleProof.PADDING_INCREMENTS : _.ceil(currentLength / CvcMerkleProof.PADDING_INCREMENTS) * CvcMerkleProof.PADDING_INCREMENTS;
     const newNodes = _.clone(nodes);
+    const secureRandom = services.container.SecureRandom;
```
Comment (jpsantosbh) [inline dist/es/creds/VerifiableCredential.js]: Same as before

File: src/uca/UserCollectableAttribute.js [hunk 4]
```diff
@@ -176,6 +172,7 @@ const parseAttestableValue = (value) => {
 function UCABaseConstructor(identifier, value, version) {
   this.timestamp = null;
   this.id = null;
+  this.secureRandom = services.container.SecureRandom;
```
Comment (jpsantosbh) [inline src/uca/UserCollectableAttribute.js]: Same as before

File: dist/cjs/uca/UserCollectableAttribute.js (hunk 2 above)
Comment (dmelosantos) [reply dist/cjs/uca/UserCollectableAttribute.js]: Same as the other comment, that pattern that leads to our error... this is an hack
The instance is an singleton, only the wire up is dynamic

**Pull request merged**